network/alsp/misbehavior.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	// UnauthorizedPublishOnChannel is a misbehavior that is reported when a message not authorized to be sent via pubsub is received via pubsub.
 	UnauthorizedPublishOnChannel network.Misbehavior = "unauthorized-pubsub-on-channel"
+
+	// PotentialSpam is a misbehavior that is reported when a message is potentially spam.
+	PotentialSpam network.Misbehavior = "potential-spam"
```

Comment:
Please remove this new type and instead use the `ResourceIntensiveRequest` type for now. We will discuss the trajectory of the development in our next sync.

Reply:
2967f4a70a86560c9265ebf56559dbf6bf66166d